src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -190,7 +186,7 @@ private String validateBody(final TransformationMode mode,
             return EMPTY_VALIDATED_BODY;
         }
 
-        if (Type.CONTENT_PACKAGE.equals(type)) {
```

Comment:
I was thought that it's better to compare actuals to statics, as statics are for sure not null. Therefore you should leave it as is.